packages/tools/src/eventDispatchers/mouseEventHandlers/mouseMove.ts

```diff
@@ -48,11 +52,38 @@ export default function mouseMove(evt: MouseMoveEventType) {
   });
 
   let annotationsNeedToBeRedrawn = false;
+  let showCrosshairsCursor = false;
 
   for (const { tool, annotations } of toolsWithAnnotations) {
     if (typeof tool.mouseMoveCallback === 'function') {
       annotationsNeedToBeRedrawn =
         tool.mouseMoveCallback(evt, annotations) || annotationsNeedToBeRedrawn;
+
+      for (const annotation of annotations) {
+        showCrosshairsCursor =
+          !!tool.getHandleNearImagePoint(
+            element,
+            annotation,
+            currentPoints.canvas,
+            6
+          ) || showCrosshairsCursor;
+      }
+    }
+  }
+
+  const showHandlesAlways = Boolean(
+    getStyleProperty('showHandlesAlways', {} as StyleSpecifier)
+  );
+
+  if (showCrosshairsCursor && showHandlesAlways) {
+    setCursorForElement(element, 'move');
+  } else {
+    const toolGroup = getToolGroupForViewport(viewportId, renderingEngineId);
+    const activeTool = toolGroup?.getActivePrimaryMouseButtonTool();
+    if (activeTool) {
+      setCursorForElement(element, activeTool);
+    } else {
+      setCursorForElement(element, 'default');
     }
   }
 
```

Comment:
I understand the rest of the changes to the show handles always, I don't understand why we are changing this file?